cms/static/js/spec/views/settings/main_spec.js

```diff
@@ -182,5 +186,69 @@ define([
             AjaxHelpers.expectJsonRequest(requests, 'POST', urlRoot, modelData);
         });
 
+        it('should save title', function(){
```

Comment:
Actually, it looked like the Bok Choy test only covered the pre-existing field-- not the new fields.